src/components/appsettings/ReposFolderPicker.vue

```diff
@@ -47,6 +44,12 @@ export default {
   components: {
     BaseIcon
   },
+  props: {
+    label: {
+      type: String,
+      default: ''
```

Comment:
If we want to allow an optional prop without a default, it can just be undefined. Empty string infers intent that isn't really needed here.
```suggestion
      default: undefined
```